### PR TITLE
Add folders and update readme

### PR DESCRIPTION
**Pull request by jashapiro**
This PR the main readme and adds additional dirs (mostly populated by `.gitkeep` files). The most substantive addition is the README in the `data` folder, which adds some thoughts about a substructure there.

Side note: I accidentally committed these changes directly to `main`, which if anybody looks we can call a "teachable moment".

Related to https://github.com/AlexsLemonade/reproducible-research/issues/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# rrp-workshop-exercises
-Materials for reproducibile research practices workshop exercises.
+# Reproducible Research Practices Workshop Exercises
+
+This repository holds materials for reproducible research workshop exercises.
+
+The directory structure serves as an example of project organization that maintains a separation among data, code and analytical results.
```
Comment (jaclyn-taroni) [inline README.md:5]:
```suggestion
The directory structure serves as an example of project organization that maintains a separation among data, code, and analytical results.
```

File: data/README.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+`data` directory organization:
+
+- `raw` contains raw files as they came from experiment results or an external source.
+These files should generally not be modified once created.
+Examples of such data are sequencing data from a core facility, expression data downloaded from a repository,
```
Comment (jaclyn-taroni) [inline data/README.txt:5]:
```suggestion
Examples of such data are sequencing data from a core facility or expression data downloaded from a repository,
```

File: data/README.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+`data` directory organization:
```
Comment (jaclyn-taroni) [inline data/README.txt:1]: Is this a TXT file on purpose? My assumption is no based on what I perceive to be Markdown formatting.

**Review by jaclyn-taroni — APPROVED** [review]
The `data/README.txt` should get fixed before this gets merged. 

In the interest of talking about what comes next, can you also comment on where you expect the Rscript (https://github.com/AlexsLemonade/reproducible-research/issues/25) and R Notebook (https://github.com/AlexsLemonade/reproducible-research/issues/27) to live? 

(I'm about to make up names that are not good but are at least somewhat illustrative of my point.)

I might naively expect that `01-preprocessing.R` and `02-viz.Rmd` to live in the root of the repository and that `analysis/` might be named`results/`. Does the data download script we use to set whatever datasets we use live in `scripts/`? Does the the script that "stitches everything together" (https://github.com/AlexsLemonade/reproducible-research/issues/29) liv in `scripts/`?

I don't think we need to get all of this sorted out here, but if you had something in mind already, it'd be good to discuss.

Going to cc: @sjspielman here since I do not expect she is watching this repo yet!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,5 @@
-# rrp-workshop-exercises
-Materials for reproducibile research practices workshop exercises.
+# Reproducible Research Practices Workshop Exercises
+
+This repository holds materials for reproducible research workshop exercises.
```
Comment (sjspielman) [inline README.md:3]:
```suggestion
This repository holds materials and exercises for the [Reproducible Research Practices Workshop](https://AlexsLemonade.github.io/reproducible-research).
```
With caveat that site is _currently_ 404.

**Review by sjspielman — APPROVED** [review]

**Pull request merged**